opentelemetry-sdk/tests/metrics/integration_test/test_cpu_time.py

```diff
@@ -98,7 +97,8 @@ def cpu_time_callback() -> Iterable[Measurement]:
                     int(states[8]) // 100, {"cpu": cpu, "state": "guest_nice"}
                 )
 
-        observable_counter = meter.create_observable_counter(
```

Comment:
As this is more of an integration test, I think it is useful to build the instruments through the meter as intended. And the meter creation methods are "more stable" so these tests are less likely to need to be modified in the future.

Reply:
All right, creating with `Meter` :v: